<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inicioPos</class>
 <widget class="QDialog" name="inicioPos">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Vertice inicial</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtX"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtY"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <item>
        <widget class="QPushButton" name="btnAceptar">
         <property name="text">
          <string>Aceptar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancelar">
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
